docs/building.md

```diff
@@ -80,8 +80,8 @@ Windows PC is setup properly.*
 ### Linux Dependencies
 
 The list of libraries and tools that must be installed on your machine in
-order to build can be found in our script
-[bootstrap-ubuntu.sh](../scripts/bootstrap-ubuntu.sh).
```

Comment:
>(../scripts/bootstrap-ubuntu.sh). [](start = 21, length = 33)

Question, if this bootstrap-ubuntu.sh got removed, then isn't that we will not have a bash file to bootstrap a linux machine during developing at local machine?

Reply:
Good question @rabbitdaxi 

I went back and forth on this. You are correct in that we will not have a bootstrap script anymore. Instead, all the dependencies are now documented in our Dockerfile. So, if you want to build locally you'll have to open up the Dockerfile and determine which packages to install based on the commands in the file.

This is not ideal, since it's not runable, but we aren't able to run the bootstrap script on our Docker images and so I'm concerned that script will get out of date since we aren't keeping it up to date.